docs/docs/components/reanimated_swipeable.md

```diff
@@ -0,0 +1,219 @@
+---
+id: reanimated_swipeable
+title: Reanimated Swipeable
+sidebar_label: Reanimated Swipeable
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import GifGallery from '@site/components/GifGallery'
+
+<GifGallery>
+  <img src={useBaseUrl("gifs/sampleswipeable.gif")} height="120" />
+</GifGallery>
+
+:::info
+This component is a drop-in replacement for the `Swipeable` component, rewritten using `Reanimated`.
+:::
+
+Reanimated `Swipeable` allows for implementing swipeable rows or similar interaction. It renders its children within a panable container allows for horizontal swiping left and right. While swiping one of two "action" containers can be shown depends on whether user swipes left or right (containers can be rendered by `renderLeftActions` or `renderRightActions` props).
+
+### Usage:
+
+To use it, import it in the following way:
+
+```js
+import Swipeable from 'react-native-gesture-handler/ReanimatedSwipeable';
+```
+
+## Properties
+
+### `friction`
+
+a number that specifies how much the visual interaction will be delayed compared to the gesture distance. e.g. value of 1 will indicate that the swipeable panel should exactly follow the gesture, 2 means it is going to be two times "slower".
+
+### `leftThreshold`
+
+distance from the left edge at which released panel will animate to the open state (or the open panel will animate into the closed state). By default it's a half of the panel's width.
+
+### `rightThreshold`
+
+distance from the right edge at which released panel will animate to the open state (or the open panel will animate into the closed state). By default it's a half of the panel's width.
+
+### `dragOffsetFromLeftEdge`
+
+distance that the panel must be dragged from the left edge to be considered a swipe. The default value is 10.
```

Comment:
```suggestion
distance that the panel must be dragged from the left edge to be considered a swipe. The default value is `10`.
```
the values should be represented as `code` - it's more readable that way

Reply:
Changed these, as well as other instances of number values.